HTML/Common.php

```diff
@@ -141,7 +141,7 @@ function _getAttrString($attributes)
         if (is_array($attributes)) {
             $charset = HTML_Common::charset();
             foreach ($attributes as $key => $value) {
-                $strAttr .= ' ' . $key . '="' . htmlspecialchars(($value ?? ''), ENT_COMPAT, $charset) . '"';
+                $strAttr .= ' ' . $key . '="' . htmlspecialchars(($value ?: ''), ENT_COMPAT, $charset) . '"';
```

Comment:
Is 0 a possible legit value here?

Reply:
Hmm - good point.  I can't say definitively but I'll account for that.